### ISSUE TITLE
WordPress website loses styling when proxied through NPM & Cloudflared Argo Tunnel

### ISSUE THREAD

**Issue by TomKamin1** [jc21/nginx-proxy-manager#1363]
**Describe the bug**
I'm getting odd behaviour when i combined Proxy manager & Cloudflared (argo tunnel) & Wordpress on my Unraid server. Instead of my website having normal styling, it seems to only display text, but ONLY on the domain, not on my LAN server ip. Cloudflare setup remains the same as it were before (when i used SWAG, it worked fine).

**Nginx Proxy Manager Version**
v2.9.8

**To Reproduce**
Correctly setup MariaDB, Cloudflared and WordPress docker templates on Unraid. Attempt to have all traffic go through cloudflared, directly to NPM.

**Expected behavior**
Check second screenshot. 

**Screenshots**
How the website looks through Nginx
![image](https://user-images.githubusercontent.com/81120446/131380296-14edb036-7380-42fa-99b8-2a46d372a1e4.png)

How it looks on LAN
![image](https://user-images.githubusercontent.com/81120446/131380331-917ad80a-52dd-4234-8f94-dca0ae8a0958.png)

**Operating System**
Unraid 6.9.2

Any help / ideas what this could be are welcome !

**Comment by mastoltejr** [comment]
I have the same issue with my pihole web UI and I have Cloudflare as my dns provider. 
![1316FB07-7C1E-41FD-ABF7-C6981652BCD7](https://user-images.githubusercontent.com/31829871/133362254-4ef10e23-f190-44dc-a00a-fe64090f3f69.jpeg)

**Comment by yun00001** [comment]
I got the same issue also. Do you guys got any resolution for this?

**Comment by jc21** [comment]
Yes wordpress likes to hardcode the entire domain url inside the database in a _lot_ of different places.

In the past I've had to run a wordpress plugin on the command line that replaces one string (domain) with another in all places and since some of those places are "php serialized" you can't just use sql for it :/

I hate wordpress for a lot of reasons but this one specifically erks me.
